inference-engine/ie_bridges/java/cpp/tensor_desc.cpp

```diff
@@ -0,0 +1,218 @@
+#include <inference_engine.hpp>
+
+#include "openvino_java.hpp"
+#include "enum_mapping.hpp"
+#include "jni_common.hpp"
+
+using namespace InferenceEngine;
+
+JNIEXPORT jlong JNICALL Java_org_intel_openvino_TensorDesc_GetTensorDesc(JNIEnv *env, jobject obj, jint precision, jintArray dims, jint layout)
+{
+    static const char method_name[] = "GetTensorDesc";
+    try
+    {
+        auto n_precision = precision_map.find(precision);
+
+        if (n_precision == precision_map.end())
+            throw std::runtime_error("No such precision value!");
+
+        auto n_layout = layout_map.find(layout);
+
+        if (n_layout == layout_map.end())
+            throw std::runtime_error("No such layout value!");
+
+        const jsize length = env->GetArrayLength(dims);
+        jint *data = env->GetIntArrayElements(dims, 0);
```

Comment:
After an `*Get*ArrayElements` we should call `Release*ArrayElements` because it allocates native memory.